taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest, HttpResponse
```

Comment:
The `HttpResponse` import is not used in this file. Consider removing it to keep the code clean.